core/src/main/java/org/elasticsearch/search/internal/ContextIndexSearcher.java

```diff
@@ -42,27 +45,43 @@
     private AggregatedDfs aggregatedDfs;
```

Comment:
hmm I guess we set the profiler later down the road but maybe we can push it up where it's needed?